plugins/org.jboss.tools.jmx.remoting1_1_3/pom.xml

```diff
@@ -0,0 +1,59 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.jboss.tools.jmx.remoting1_1_3</groupId>
+	<artifactId>org.jboss.tools.jmx.remoting1_1_3</artifactId>

```

Comment:
want to include all binaries that we use so that they all get fetched and copied via maven dependency plugin?